### ISSUE TITLE
Implement new Grid with Photo Gallery Page

### ISSUE THREAD

**Issue by HarveyPeachey** [bbc/simorgh#7520]
**Is your feature request related to a problem? Please describe.**
**Parent Issue** https://github.com/bbc/simorgh/issues/3686

To break down the work for migrating over to our new grid implementation, we should focus on doing this on a page type basis with this issue focusing on `PhotoGalleryPage`.

**Describe the solution you'd like**
To be able to migrate over to the new grid, we need to replace all usage of [`GridItemConstrained`](https://github.com/bbc/simorgh/blob/latest/src/app/lib/styledGrid/index.jsx) in our containers to make use of the new grid. The containers used on `PhotoGalleryPage` are as follows:

- FauxHeadline (Shared with #7520)
- VisuallyHiddenHeadline (Shared with #7520)
- Headings (Shared with #7520 and #7518)
- CpsText (Shared with #7520)
- Image (Shared with #7520 and #7518)
- ArticleTimestamp (Shared with #7520 and #7518)
- CpsAssetMediaPlayer - Not sure why PGL's have this, as I'm pretty sure they don't have videos

We need to check if these containers are using the old grid and if so, we need migrate them over to the new implementation. Once the containers have been migrated we can then implement the new grid into `PhotoGalleryPage`. Some containers are shared with other page types, ensure that if those issues have been picked, that work isn't being duplicated. 

This issue and the issue for [MAP's](https://github.com/bbc/simorgh/issues/7519) have exactly the same containers used for blocks, so when work is carried out on either one, this should also complete the work for both.

A developer accessiblity swarm should be carried out on a PGL when the new grid has been implemented before going live.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing. Regression checks are needed.
- Update snapshot tests

**Additional context**
